booking/src/main/java/ru/org.dsr/search/SearchKinopoiskJSOUP.java

```diff
@@ -0,0 +1,192 @@
+        String date = e.select("span.date").text().replace('\"', '\'');
+        String author = e.select("div > div > p.profile_name > a").text().replace('\"', '\'');
+        return createComment(author, title, desc, date, SITE);
+    }
+
+    private Item initBook() throws RequestException, RobotException {
+        Document pageBook = getDoc(URL_MAIN_ITEM);
+
+        String desc = null, firstName, lastName, urlImg;
+
+        try {
+            desc = getDesc(pageBook);
+        } catch (LoadedEmptyBlocksException e) {
+            log.warn(e.toString());
+        } catch (NoFoundElementsException e) {
+            log.warn(e.toString());
+        }
+
+        try {
+            firstName = getFirstName(pageBook);
+        } catch (LoadedEmptyBlocksException e) {
+            log.warn(e.toString());
+            firstName = itemID.getFirstName();
+        } catch (NoFoundElementsException e) {
+            log.warn(e.toString());
+            firstName = itemID.getFirstName();
+        }
+
+        try {
+            lastName = getLastName(pageBook);
+        } catch (LoadedEmptyBlocksException e) {
+            log.warn(e.toString());
+            lastName = itemID.getLastName();
+        } catch (NoFoundElementsException e) {
+            log.warn(e.toString());
+            lastName = itemID.getLastName();
+        }
+
+        try {
+            urlImg = getUrlImg(pageBook);
+        } catch (LoadedEmptyBlocksException e) {
+            log.warn(e.toString());
+            urlImg = "";
+        } catch (NoFoundElementsException e) {
+            log.warn(e.toString());
+            urlImg = "";
+        }
+        return new Item(new ItemID(firstName, lastName, "MOVIE"), desc, urlImg);
+    }
+
+    private String getFirstName(Document document) throws NoFoundElementsException, LoadedEmptyBlocksException {
+        Elements elsFirstName = document.select("#headerFilm > h1 > span.moviename-title-wrapper");
+        if (elsFirstName == null || elsFirstName.isEmpty()) {
+            throw new NoFoundElementsException(URL_MAIN_ITEM, "#headerFilm > h1 > span.moviename-title-wrapper");
+        }
+        String firstName = elsFirstName.get(0).text();
+        if (firstName == null || firstName.isEmpty())
+            throw new LoadedEmptyBlocksException("#text", "#headerFilm > h1 > span.moviename-title-wrapper", URL_MAIN_ITEM);
+        return firstName;
+    }
+
+    private String getLastName(Document document) throws NoFoundElementsException, LoadedEmptyBlocksException {
+        Elements elsLastName = document.select("#infoTable > table > tbody > tr:nth-child(6) > td:nth-child(2) > a");
+        if (elsLastName == null || elsLastName.isEmpty()) {
+            throw new NoFoundElementsException(URL_MAIN_ITEM, "#infoTable > table > tbody > tr:nth-child(6) > td:nth-child(2) > a");
+        }
+        String lastName = elsLastName.text();
+        if (lastName == null || lastName.isEmpty())
```

Comment:
Better to implement your own function isNullOrEmpty. Or use some lib like `guava` which already contain a lot of great helper functions.
PS: There is StringUtils.isEmpty() in spring-core.